charts/linkerd-control-plane/Chart.yaml

```diff
@@ -16,7 +16,7 @@ dependencies:
 - name: partials
   version: 0.1.0
   repository: file://../partials
-version: 1.15.0
+version: 1.16.0-edge
```

Comment:
We haven't branched out `release/stable-2.14`, so there's no need to bump the minor segment here, unless you think changes were big enough to justify a minor bump.

Reply:
ah :( I did a minor bump for all of them based on the release instructions. Can do patch bumps for all of them except this one. We added policy resource fields so it might be worth bumping the minor. wdyt @alpeb